test/test_decomp.py

```diff
@@ -1164,7 +1163,7 @@ def test_weight_norm_interface(self, device):
         [
             xfail(
                 "nn.functional.scaled_dot_product_attention",
-                dtypes=[torch.half] + ([torch.bfloat16] if IS_MACOS else []),
```

Comment:
is there a reason why removed the bfloat16 test?

Reply:
This is for *adding* BF16 for MacOS test. Originally BF16 operator cannot run, maybe because we don't have BF16 support on MacOS, so explicitly assert failure with *xfail*. Now with this PR, we can run BF16 operator since inside the op it uses FP32. So we don't need to assert *xfail* for BF16 on MACOS.